katas/7kyu/two-to-one.js

```diff
@@ -0,0 +1,8 @@
+function longest(s1, s2) {
+    const s1Set = new Set(s1.split(''));
+    const s2Set = new Set(s2.split(''));
```

Comment:
it'd be better to combine two string to call the new Set function only once.
it's not really critical at this point but you have to always remember that you have to avoid using "expensive" functions during development more than it's really necessary.
so, at this point you can refactor it
```js
const set = new Set((s1 + s2).split(''))
```